### ISSUE TITLE
README doesn't explain how to install json-server

### ISSUE THREAD

**Issue by combinatorist** [egeehartz/gatherings-app#39]
In step 6 of the [Setup](https://github.com/egeehartz/gatherings-app/blob/master/README.md#setup), it says to use `json-server`, but I don't have that installed. Could that be in package.json or does it have to be a separate global install?

**Comment by combinatorist** [comment]
... but maybe I don't need that. I was able to skip to step 7 `npm start` and (eventually) gave me a localhost webpage, that looks functional. Maybe I wouldn't run into problems until later.

**Comment by combinatorist** [comment]
Yeah, it freezes when I try to "sign in", I'm guessing because it doesn't have the json server to take in my info or show the next page or something.

**Comment by combinatorist** [comment]
Same even if I try to register as a new member.

**Comment by combinatorist** [comment]
But, I was able to get it running by doing `npm --global install json-server` first. I actually prefer not to globally install npm packages, so then I did this to keep it separate from other projects:
```
npm install json-server
cd api
../node_modules/json-server/lib/cli/bin.js -w database.json -p 8088
```

Which is wonky, but you could probably make a project script access (called something like 'start-db') or there may be a better way to use project dependencies directly in npm - IDK.

**Comment by combinatorist** [comment]
Haven't tested this, but maybe it would work: https://firstdoit.com/no-need-for-globals-using-npm-dependencies-in-npm-scripts-3dfb478908.

**Comment by combinatorist** [comment]
Also seems from that post, that I could have used the much simpler path: `node_modules/.bin/json-server` to run locally. I learned a lot tonight! :)

**Comment by egeehartz** [comment]
Thanks for checking it out! I'll look at this more in the morning and hopefully have an updated README later this week.

**Comment by egeehartz** [comment]
I updated the instructions , is that more clear now?

**Comment by combinatorist** [comment]
Yup, definitely! Now it says to install json-server. I suspect, though that it may need to be a global install in order to get the command line (something that really bothers me about npm CLI's), but you may also want to leave that up to personal preference, IDK.

**Comment by combinatorist** [comment]
Oh, and I have hints for other users about using it locally without a global install above. But, like I said, that's probably up to personal preference.
